core/libp2p/protocol/identify/identify_delta.cpp

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "libp2p/protocol/identify/identify_delta.hpp"
+
+#include <string>
+#include <unordered_set>
+
+#include "libp2p/basic/protobuf_message_read_writer.hpp"
+#include "libp2p/protocol/identify/utils.hpp"
+
+namespace {
+  const std::string kIdentifyDeltaProtocol = "/p2p/id/delta/1.0.0";
```

Comment:
maybe put such protocols tags into class like IdentifyDelta:: kIdentifyDeltaProtocol. Otherwise you need to copy these strings in test, which is as bad as any other copying of code

Reply:
Method `getProtocolId()` is used to retrieve those strings, and it also can be used in tests, but the purpose of testing is to check if that string (and thus a protocol) corresponds to the expected one. So, copying strings here is absolutely necessary IMHO

Reply:
There's no difference between storing the string here or directly returning it